### DISCUSSION TITLE
where are native downloads located ?

### DISCUSSION THREAD

**Discussion by avichou** [pear-devs/pear-desktop#2160] [Q&A]
using native youtube music downloads (whicch was recently enabled by google ) , where can we find donwload location ? i want to exclude them from time machine backup since it takes a lot of space 

<img width="688" alt="image" src="https://github.com/th-ch/youtube-music/assets/86660671/6fb67915-ac6b-4526-a020-c9d732680c13">

**Comment by TecnikOfficial** [comment]
downloaded songs using native download button are typically stored within the web app itself and not directly accessible from your device's file system. but u can choose download folder using plugins option 
<details>
  <summary>Click to view screenshot</summary>
  <img src="https://github.com/user-attachments/assets/50573bd2-22eb-40af-b40c-c60dd9f9a35e" alt="Screenshot">
</details>

**do note : currently download option is not working due to some changes implement by youtube so dependency update is needed . wait for update from dev** :) #2208
